README.md

```diff
@@ -1 +1,51 @@
 [Blade](https://github.com/javan/blade) plugin for [Sauce Labs](https://saucelabs.com/)
+
+## Configuration
+
+### Authenticate with Sauce Labs
+
+Set the `SAUCE_USERNAME` and `SAUCE_API_KEY` environment variables to authenticate with Sauce Labs.
+
+All CI tools provide a way to set environment variables for a test run. For non-CI test runs, set the environment variables in your shell or in your test runner script.
```

Comment:
Can pass credentials in `.blade.yml` also, but figure we should focus on the preferred way to provide them (externally, not embedded in code)